### DISCUSSION TITLE
Linker issues

### DISCUSSION THREAD

**Discussion by fekuxle** [ocornut/imgui#5314] [New Users Build/Link/Run issues ONLY!]
Been trying to get a minimal imgui project to compile for a few days now, but i can't get anything to work
I added all required source files to my project, yet it never gets past linking
I tried multiple different setups, even changing the solution files manually to include the source files
I'm aware this technically isn't an imgui problem and more of a build system problem, but i haven't had this problem anywhere else
I am currently just using the code from the win32 dx10 example, and it works fine using the example directly but not in my own project with that code just pasted in

Any ideas on how to get this to link correctly?
Can provide my testing project in a git repo or something if needed

**Comment by rokups** [comment]
You did not describe issue you are having. "Linker issues" is way too abstract for anyone to be able to help you. Compiler error messages would be a good start. :+1:

**Comment by monicaafilip** [comment]
I recommend sharing the output of the build, so that we can help you with the exact linking error.
Also you should try looking into a tutorial and go step by step.
This was the way I first managed to work with the IMGUI project.
Here is a good tutorial I watched : https://youtu.be/VRwhNKoxUtk

**Reply by fekuxle** [reply]
i did end up getting it to work in the end by manually editing the solution files to include all the neccesary files, but the way that tutorial does it is way easier
thanks
